hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -81,6 +121,14 @@ class NavBar extends React.Component<INavBarProps> {
             <span>Disk Usage</span>
             <Link to='/DiskUsage'/>
           </Menu.Item>
+          {
+            this.state.isHeatmapAvailable ?
+              <Menu.Item key='/HeatMap'>
+                <Icon type='bar-chart' />
+                <span>Heatmap</span>
+                <Link to='/HeatMap' />
```

Comment:
if you want to do the rename in this PR
```suggestion
                <Link to='/Heatmap' />
```